### ISSUE TITLE
python main.py --full true

### ISSUE THREAD

**Issue by zcswdt** [YoongiKim/AutoCrawler#32]
Hello, I can download thumbnails using `python main.py`, but can’t download with parameter `python main.py --full true`.Please help me, thanks！
![image](https://user-images.githubusercontent.com/43515926/104871970-8151a280-5987-11eb-9ca1-ff918f076478.png)

**Comment by YoongiKim** [comment]
Hello.

It seems running ok in screenshot.
But in full mode, no_gui option is true because it is faster.
If you want to see the running process, please try ```python main.py --full true --no_gui false```

Please add comment if you have more issues.
Thank you.

**Comment by zcswdt** [comment]
> you want to see the running process,

The problem has been solved, thank you very much for your help and patient answers.

**Comment by Ivan-Fan** [comment]
Hi! Thanks a lot for your great work! I wanted to download images in full mode and I test the code as above and set the picture limit as 10. However, I found the image number becoming far bigger than the limit I set (shown as below). I was wondering if anything was wrong?
<img width="734" alt="1" src="https://user-images.githubusercontent.com/39240631/106439979-41c0a580-64b3-11eb-8d47-c1cd97b5f170.png">
